netlify.toml

```diff
@@ -2,15 +2,16 @@
   # Directory (relative to root of your repo) that contains the deploy-ready
   # HTML files and assets generated by the build. If a base directory has
   # been specified, include it in the publish directory path.
-  publish = "docs/export/"
+  publish = "export/"
+  base = "docs/"
 
   # Default build command.
-  command = "pnpm docs:build"
+  command = "pnpm build"
 
 [build.environment]
   NODE_VERSION = "18"
   NODE_OPTIONS = "--max_old_space_size=4096"
   PNPM_FLAGS = "--shamefully-hoist"
 
 [[plugins]]
-  package = "./node_modules/@mui/monorepo/packages/netlify-plugin-cache-docs"
```

Comment:
🤔 We should try the same trick as we do with `eslint-plugin-material-ui`

Reply:
I've tried a minimal setup with the main requirements—manifest file and re-exporting the `@mui/monorepo` package.
WDYT? 🤔